tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_bluetooth.py

```diff
@@ -33,7 +32,10 @@ def test_toggle_bluetooth_settings(self):
 
         self.assertEquals(bluetooth_settings.device_name, device_name)
 
-        bluetooth_settings.enable_visible_to_all()
+        bluetooth_settings.enable_visible_to_all(self.data_layer)
```

Comment:
This is weird, I would do in the enable_visible_to_all function just do the checking of the checkbox that it is checked (wait().until checked) and then do an assert about self.data_layer.bluetooth_is_discoverable is True (and Johan would probably not agree with that part).